safe-global/web-core
# feat: redesign address book import

## What it solves
Our import dialogs have inconsistent designs

Resolves #1118 

## How this PR fixes it
- Extracts `FileInput` from `ImportAllDialog` 
- uses this component in `ImportDialog` of addressbook

## How to test it
Open the import dialog in addressbook
It should work the same but have the same design as our global Import All dialog

## Analytics changes
None

## Screenshots
![Screenshot 2022-11-14 at 13 54 26](https://user-images.githubusercontent.com/2670790/201665880-6a74e78a-1e2a-4814-85fa-75a0b8216bdd.png)
![Screenshot 2022-11-14 at 13 56 44](https://user-images.githubusercontent.com/2670790/201665875-59a67deb-a55a-408e-a9b7-0cf835c119b9.png)
